.github/workflows/integration-tests.yml

```diff
@@ -27,6 +27,25 @@ jobs:
       fail-fast: false
 
     services:
+      frankenphp:
```

Comment:
reordering because apparently frankenphp is slow to start so it'd benefit from booting it first